app/viewmodels/CheckYourAnswersHelper.scala

```diff
@@ -82,20 +82,20 @@ class CheckYourAnswersHelper(userAnswers: UserAnswers, affinityType: AffinityTyp
   def hasSecondContactPage(): Option[SummaryListRow] = {
     val summaryView = (yesNo: String) =>
       SummaryListRowViewModel(
-        key = "hasSecondContact.checkYourAnswersLabel",
+        key = "haveSecondContact.checkYourAnswersLabel",
         value = ValueViewModel(HtmlFormat.escape(s"${messages(yesNo)}").toString),
         actions = Seq(
-          ActionItemViewModel("site.change", routes.IndexController.onPageLoad().url)
+          ActionItemViewModel("site.change", routes.HaveSecondContactController.onPageLoad().url)
             .withAttribute(("id", "second-contact"))
             .withVisuallyHiddenText(" if you have a second contact")
```

Comment:
This should be in the message file

Reply:
This should be fixed now for all of them